entries/entry03.md

```diff
@@ -1,8 +1,75 @@
+
+
 # Entry 3
```

Comment:
Sources: 1
EDP: 1
Knowledge: 0 (use code snippets and screenshots as evidence of your learning!)
Skills: 1
Professionalism: 1